src/crimson/os/seastore/transaction.h

```diff
@@ -150,6 +150,7 @@ class Transaction {
       assert(read_set.count(ref->prior_instance->get_paddr()));
       ref->reset_prior_instance();
     } else {
+      assert(ref->is_stable_written());
```

Comment:
?Why drop this assert?

Reply:
This is added, not dropped.

Reply:
Ah, sorry, my brain was working properly:-)

Reply:
No worries.